webui/src/pages/sidepanel/sidepanel.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { FunctionComponent } from 'react';
```

Comment:
Add license header comment.

This file should go to `src/components` rather than `src/pages`, right?